tests/unit/replicator_tests.py

```diff
@@ -116,6 +118,7 @@ def test_replication_with_generated_id(self):
         clone = Replicator(self.client)
         clone.create_replication(self.db, self.target_db)
 
+    @flaky(max_runs=3)
```

Comment:
N is set to 1, so I doubt tweaking the values some more will make a difference tbh. @smithsz thoughts?